TL improvement & Official TL import (#374)
07th-mod/higurashi-console-arcs: Update/omot_021.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 5, "ps3/s14/05/220500203", 256, TRUE);
	OutputLine(NULL, "「鷹野は、その黒幕に自分たちの正当性を認めさせることで、緊急マニュアルにもとづく非常行動を起こすことができるのです」",

> 			NULL, "\"Takano can only take action according to the emergency manual if only the mastermind's authorizes her action.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Takano can only take action according to the emergency manual as long as the mastermind authorizes her actions.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 5, "ps3/s14/05/220500205", 256, TRUE);
	OutputLine(NULL, "「それは、鷹野が自分の理論を正当化させる目的を持っていることから考えても、絶対に回避するはずなのですよ」",

> 			NULL, "\"Considering that her motivation is to justify her theories, that's why Takano wants to head off it at any cost.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Considering that Takano's motive is to justify her theories, she'd want to avoid that at all costs.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300095", 256, TRUE);
	OutputLine(NULL, "「ただ、赤坂さんの役割はバックボーンの調査みたいだから、梨花ちゃんの防衛とは切り離して考えないと。",

> 			NULL, "\"However, it seems that Akasaka-san's role in this operation is to investigate the backbone of them, so we'll have to set him aside when it comes to Rika-chan's defense.", GetGlobalFlag(GLinemodeSp));

Reviewer: ```suggestion
			NULL, "\"However, it seems that Akasaka-san's role in this operation is to investigate the enemy's backbone, so we'll have to set him aside when it comes to Rika-chan's defense.", GetGlobalFlag(GLinemodeSp));
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300096", 256, TRUE);
	OutputLine(NULL, "それと重要なのは、今行方不明になっている富竹さんだよ」",

> 			NULL, " And more important topic is about Tomitake-san, who is still unaccounted for.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " And the more important subject is Tomitake-san, who is still unaccounted for.\"", Line_Normal);
```

			NULL, "\"I think the possibility that he was killed is low.", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300098", 256, TRUE);
	OutputLine(NULL, "もしも死んでいたら、鉄平じゃなくて富竹さんが祟りの犠牲として発見されていたはずだからね」",

> 			NULL, " Because If Tomitake-san were dead, he would've been discovered as the victim of the curse, instead of Teppei.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Because if Tomitake-san were dead, he would've been discovered as the victim of the curse, instead of Teppei.\"", Line_Normal);
```

			NULL, "\"Incidentally, I explained the situation to Ooishi when we came here last night.", Line_WaitForInput);
	ModPlayVoiceLS(4, 5, "ps3/s14/05/220500208", 256, TRUE);
	OutputLine(NULL, "赤坂もフォローしてくれましたが、まだ迷っているみたいなのです」",

> 			NULL, " Even though Akasaka helped me to explain, Ooishi still seemed conflicted.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Even though Akasaka helped me explain, Ooishi still seemed conflicted.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#c89a80>入江</color>", NULL, "<color=#c89a80>Irie</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 10, "ps3/s14/10/22100262", 256, TRUE);
	OutputLine(NULL, "「すみません、…『東京』との繋ぎは全て富竹さんと鷹野さんにお任せしていたので、私には良い手段が…」",

> 			NULL, "\"I'm sorry... I left all contact with \"Tokyo\" in the hands of Tomitake-san and Takano-san, so I don't come up with anything...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"I'm sorry... I left all contact with \"Tokyo\" in the hands of Tomitake-san and Takano-san, so there's little I can do there...\"", Line_Normal);
```

//rしかし、こんな敵陣の中で協力してもらえるだけでもありがたいのだし、それにこの人が一番大変になるのは、むしろ全てがうまくいって終わったときだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "しかし、こんな敵陣の中で協力してもらえるだけでもありがたいのだし、それにこの人が一番大変になるのは、むしろ全てがうまくいって終わったときだ。",

> 			NULL, "But I was thankful to have anyone from the enemy camp on my side. And this man will go through the hardest time if everything ends well.", Line_Normal);

Reviewer: ```suggestion
			NULL, "But I was thankful to have anyone from the enemy camp on my side. And this man will go through the hardest time even if everything ends well.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f0953d>レナ</color>", NULL, "<color=#f0953d>Rena</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 2, "ps3/s14/02/220200069", 256, TRUE);
	OutputLine(NULL, "防衛だけならまだしも、第二波第三波と攻められたら、やっぱり弱いよ」",

> 			NULL, " It'll be easy if all we have to do is defend ourselves, but it's hard to fight off them in a second or third wave.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " It'll be easy if all we have to do is defend ourselves, but it'll be hard to fight off a second or third wave.\"", Line_Normal);
```

//rそして俺たちは、周囲の様子をさりげなくうかがってから泣き崩れる格好で顔を彼に近づけると、ぼそぼそ、と伝えるべきことを伝えた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして俺たちは、周囲の様子をさりげなくうかがってから泣き崩れる格好で顔を彼に近づけると、ぼそぼそ、と伝えるべきことを伝えた。",

> 			NULL, "Then, we casually looked at our surroundings, before pretending to collapse into tears and approaching him, and we quietly whispered in his ear what we needed to tell.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Then, we casually looked at our surroundings, before pretending to collapse into tears while approaching him, and we quietly whispered what we needed to say into his ear.", Line_Normal);
```

//rそれも覚悟の上ってやつですか。大したもんだ、さすがは園崎家の片割れだけはある。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それも覚悟の上ってやつですか。大したもんだ、さすがは園崎家の片割れだけはある。",

> 			NULL, "So she's even prepared for that? What quite a girl. She really is a member of the Sonozaki family.", Line_Normal);

Reviewer: ```suggestion
			NULL, "So she's even prepared for that? That's impressive. She really is a member of the Sonozaki family.", Line_Normal);
```

//rしくじった。…鷹野は臍をかむ思いで、自分のやりすぎを後悔した。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "しくじった。…鷹野は臍をかむ思いで、自分のやりすぎを後悔した。",

> 			NULL, "Takano felt they had totally screwed up. ...She was completely overwhelmed with regret of overagitating.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Takano felt they had totally screwed up. ...She was completely overwhelmed with regret for overreaching.", Line_Normal);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#3e5745>小此木</color>", NULL, "<color=#3e5745>Okonogi</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 14, "ps2/14/223400015", 540, TRUE);
		OutputLine(NULL, "「三佐、…あるいは、という仮定をお許し願えるなら申し上げますが。",

> 				NULL, "\"Major... forgive me for being presumptuous to speak about what-if, but...", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, "\"Major... forgive me for being presumptuous, but...", Line_WaitForInput);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#3e5745>小此木</color>", NULL, "<color=#3e5745>Okonogi</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 14, "ps3/s14/14/vt4bc_okono1024", 256, TRUE);
		OutputLine(NULL, "「三佐、…あるいは、という仮定をお許し願えるなら申し上げますが。",

> 				NULL, "\"Major... forgive me for being presumptuous to speak about what-if, but...", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, "\"Major... forgive me for being presumptuous, but...", Line_WaitForInput);
```

		ModPlayVoiceLS(4, 14, "ps3/s14/14/vt4bc_okono1025", 256, TRUE);
		OutputLine(NULL, "Ｒは、園崎詩音の手ですでに殺害されていると判断して…作戦を予定通りに進めてもよろしいのではないでしょうか？」",

> 				NULL, " It's okay that we assume that R has already been killed by Shion Sonozaki... and proceed the operation as planned, isn't it?\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " It's okay to assume that R has already been killed by Shion Sonozaki... and proceed with the operation as planned, isn't it?\"", Line_Normal);
```

		ClearMessage();

	//小此木rvS14/14/VT4Bc_okono1026.「隊員たちの報告から判断しても、その扱いで進めて問題はないと私は考えますが」
		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#3e5745>小此木</color>", NULL, "<color=#3e5745>Okonogi</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 14, "ps3/s14/14/vt4bc_okono1026", 256, TRUE);
		OutputLine(NULL, "「隊員たちの報告から判断しても、その扱いで進めて問題はないと私は考えますが」",

> 				NULL, "\"Judging from the information we got from the team pointed to that, I don't think there's any problem with continuing it.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, "\"Judging by the reports from the team, I don't think there's any problem with continuing.\"", Line_Normal);
```

				NULL, " Although, that fool Ootaka may come here eventually... keep that in mind, just in case.", Line_WaitForInput);
		ModPlayVoiceLS(4, 11, "ps3/s14/11/vt4bc_oishi1041", 256, TRUE);
		OutputLine(NULL, "あぁもうこんな時間、まだ残ってくれているといいんだけど…」",

> 				NULL, " Aah, look at the time! I hope there's still him left there...\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " Aah, look at the time! I hope he's still there...\"", Line_Normal);
```
